src/Akka.Streams.Kafka/Settings/ConsumerSettings.cs

```diff
@@ -160,6 +161,14 @@ public static ConsumerSettings<TKey, TValue> Create(Akka.Configuration.Config co
         /// Helpful for debugging, but do not recommend running in production with this enabled.
         /// </remarks>
         public bool VerboseLogging { get; init; } = false;
+
+        /// <summary>
+        /// Maximum number of records to fetch per poll attempt.
+        /// </summary>
+        /// <remarks>
+        /// Defaults to 10.
+        /// </remarks>
+        public int MaxPollRecords { get; init; } = 10;
```

Comment:
Allow the end-user to specify the maximum number of records polled as a tuning parameter - this is something you can set directly inside the Java Kafka client but not .NET's due to some design differences, so we're kind of simulating that here.